pkg/controller/operators/operatorconditiongenerator_controller.go

```diff
@@ -34,9 +37,35 @@ func (r *OperatorConditionGeneratorReconciler) SetupWithManager(mgr ctrl.Manager
 		IsController: true,
 		OwnerType:    &operatorsv1alpha1.ClusterServiceVersion{},
 	}
+	p := predicate.Funcs{
+		CreateFunc: func(e event.CreateEvent) bool {
+			if _, ok := e.Meta.GetLabels()[operatorsv1alpha1.CopiedLabelKey]; ok {
+				return false
+			}
+			return true
+		},
+		DeleteFunc: func(e event.DeleteEvent) bool {
+			if _, ok := e.Meta.GetLabels()[operatorsv1alpha1.CopiedLabelKey]; ok {
+				return false
+			}
+			return true
+		},
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			if _, ok := e.MetaOld.GetLabels()[operatorsv1alpha1.CopiedLabelKey]; ok {
+				return false
+			}
+			return e.ObjectOld != e.ObjectNew
```

Comment:
Does `!=` have the right semantics for this?

Reply:
I had copied this bit from the [example code](controller ) in kubebuilder, I'm fine returning true though.

Reply:
Yeah, I suppose even if for whatever reason you have two value-equal objects for which `!=` evaluates to false, nothing bad happens. Returning `true` should be practically the same, and might save people like me from pausing to think about it, but I'm good with it either way. Thanks!

Reply:
In the interest of improved clarity I am updating this return statement to always return true.